### PR TITLE
Update changelog - Robusta Beans Sprint 1

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Mailet DOC: Exclude from documentation annotation thanks to [mschnitzler](https://github.com/mschnitzler)
 - `cassandra.pooling.max.queue.size` configuration option Thanks to [matzepan](https://github.com/matzepan)
 - `RecipentDomainIs` and `SenderDomainIs` matchers by [athulyaraj](https://github.com/athulyaraj)
+- Metrics for BlobStore
+- New Guice product using Cassandra RabbitMQ ElasticSearch and Swift (experiemental)
```
Comment (trantienduchn) [inline CHANGELOG.md]: and one more ldap product? 
s/Swift/Openstack Swift? to avoid miss understanding

**Comment by chibenwa** [comment]
> could you add ReIndexing API changes?

I keep this on purpose for when https://github.com/linagora/james-project/pull/1982 would be merged

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,8 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
- 
-## [3.2.0] - 2018-11-XX
+
+### Added
+- Metrics for BlobStore
+- New Guice product using Cassandra RabbitMQ ElasticSearch, OpenStack Swift and optional LDAP dependency (experiemental)
```
Comment (mbaechler) [inline CHANGELOG.md]: you didn't fixed the type on experimental

Comment (chibenwa) [reply CHANGELOG.md]: True, sorry.

**Comment by chibenwa** [comment]
> Do you mind to update it, or do you prefer I do it in an other PR?

Let's do that here!

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ### Changed
 - WebAdmin ReIndexing API had been reworked
+- MailboxListener and mailbox event system was reworked. Custom listeners will need to be adapted. Please see Upgrade instuctions.
```
[the remark this answers is not in the document]
Comment (rouazana) [reply CHANGELOG.md]: fixup pushed

**Comment by rouazana** [comment]
tess this please

**Comment by rouazana** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**